### PR TITLE
Action subgenerator

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: generators/action/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    this.argument('moduleName', {type: String, required: true});
+    this.argument('actionName', {type: String, required: true});
+  },
+
+  initializing: {
+    paths: function() {
+      fs.statSync(path.join(this.config.get('modulesPath'), this.moduleName), function(err) {
+        if (err) throw err;
+      })
+
```
Comment (ghost) [inline generators/action/index.js:20]: _To ensure that module is not automatically created if it does not exist_

Note to: @imranismail

File: generators/action/templates/action.js.ejs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const <%= actionName %> = () => ({
+  type: types.<%= actionName.toUpperCase() %>,
+})
+
```
Comment (ghost) [inline generators/action/templates/action.js.ejs:4]: _consider to snake case this?_

Note to: @imranismail

File: generators/action/templates/types.js.ejs [hunk 3]
```diff
@@ -0,0 +1 @@
+export const <%= actionName %> = `${NAME}/<%= actionName.toUpperCase() %>`
```
Comment (ghost) [inline generators/action/templates/types.js.ejs]: _snake case this too?_

Note to: @imranismail

File: generators/action/templates/action.js.ejs (hunk 2 above)
Comment (imranismail) [inline generators/action/templates/action.js.ejs:4]: this is for action right? if yes, ya.. must camelCase.. maybe we can include lodash in the deps to do this @fardormu

File: generators/action/templates/types.js.ejs (hunk 3 above)
Comment (imranismail) [inline generators/action/templates/types.js.ejs]: actionName.toUpperCase() for types will suffice @fardormu

File: generators/action/index.js (hunk 1 above)
Comment (imranismail) [inline generators/action/index.js:20]: Sweet

**Pull request merged**